### DISCUSSION TITLE
backups and logs not working!!

### DISCUSSION THREAD

**Discussion by tekjunkie6502** [mealie-recipes/mealie#2998] [General]
Sorry really noob here and a bit lost..
I use this https://mariushosting.com/how-to-install-mealie-v1-on-your-synology-nas/ guide and it works 
I used to have 1.0xbeta and made a backup.
Then copied the directory and and started a blank.

I now have the nighty and i only see empty logs and the backups don't restore.
I have tried to upload an old backup and doesn't do anything.
So don't even get a way to restore.

Been runing for about 9 months so have a very nice list so really want this to work...
So got lots of backups and the complete docker folder.

any help forward?

**Comment by Kuchenpirat** [comment]
Hi, 
if you have used the guide from mariushosting you have to drop the `read_only: true` in your docker-compose.yml file. 

This prevents mealie from creating an temporary location for the backup. After that you should be good to go.

**Comment by tekjunkie6502** [comment]
Yep that was it, thanks you very much.

**Reply by Kuchenpirat** [reply]
No worries. Have fun with mealie
